From 8e62087392d0bdfe3d119d07df5fea42a419edfb Mon Sep 17 00:00:00 2001
From: Vendula Poncova <vponcova@redhat.com>
Date: Thu, 14 Sep 2017 13:40:54 +0200
Subject: [PATCH] Don't setup the hub twice (#1491333)

We need to make sure, that the simpleline's setup is called as well,
so simpleline will remember that the setup was already done.

Resolves: rhbz#1491333
---
 pyanaconda/ui/tui/hubs/__init__.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/pyanaconda/ui/tui/hubs/__init__.py b/pyanaconda/ui/tui/hubs/__init__.py
index 616b40bf6..90af386c3 100644
--- a/pyanaconda/ui/tui/hubs/__init__.py
+++ b/pyanaconda/ui/tui/hubs/__init__.py
@@ -62,6 +62,7 @@ class TUIHub(TUIObject, common.Hub):
         self.input_required = True
 
     def setup(self, args="anaconda"):
+        TUIObject.setup(self, args)
         environment = args
         cats_and_spokes = self._collectCategoriesAndSpokes()
         categories = cats_and_spokes.keys()
-- 
2.13.5

